sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/aio/_digitaltwins_client_async.py

```diff
@@ -573,11 +587,11 @@ async def list_event_routes(self, **kwargs):
 
     @distributed_trace_async
     async def upsert_event_route(self, event_route_id, event_route, **kwargs):
-        # type: (str, "models.EventRoute", **Any) -> None
+        # type: (str, ~azure.digitaltwins.models.DigitalTwinsEventRoute, **Any) -> None
```

Comment:
This syntax (~azure.digitaltwind.DigitalTwinsEventRoute) should only be used in the docstrings.
For the type hints, you can use the imported model directly.

Reply:
Fixed.